scripts/download_hdx_admin_boundaries.sh

```diff
@@ -28,32 +28,35 @@ if wget -O $ckan_package_json_path "https://data.humdata.org/api/3/action/packag
     do
         file_extension=${filename##*.}
         mkdir -p "data/out/country_extractions/"$country_code"/202_admn/"
-        cp $filename "data/out/country_extractions/"$country_code"/202_admn/"$country_code"_admn_ad0_py_s4_unocha_pp_adminboundary0."$file_extension
+        display_name=$(cat static_data/admin_level_display_names.json | jq --raw-output ".${country_code}.adm0 // \"adminboundary0\"")
```

Comment:
do we need to fill values for every country in **admin_level_display_names.json**?

Reply:
no, `//` is to fallback to `adminboundary0`, `adminboundary1`, `adminboundary2` for `adm0`, `adm1` and `adm2`